grakn-graph/src/main/java/ai/grakn/factory/GraknGraphFactoryImpl.java

```diff
@@ -157,21 +156,17 @@ static ConfiguredFactory configureGraphFactory(String keyspace, String location,
      * @return A new or existing grakn graph factory with the defined name connecting to the specified remote location
      */
     private static ConfiguredFactory configureGraphFactoryRemote(String keyspace, String engineUrl, String graphType){
-        try {
-            String restFactoryUri = engineUrl + GRAPH_FACTORY_URI + "?" + GRAPH_CONFIG_PARAM + "=" + graphType;
-
-            Properties properties = new Properties();
-            properties.load(new StringReader(EngineCommunicator.contactEngine(restFactoryUri, REST.HttpConn.GET_METHOD)));
+        String restFactoryUri = engineUrl + GRAPH_FACTORY_URI + "?" + GRAPH_CONFIG_PARAM + "=" + graphType;
 
-            String computer = null;
-            if(properties.containsKey(COMPUTER)){
-                computer = properties.get(COMPUTER).toString();
-            }
+        Properties properties = new Properties();
+        properties.putAll(read(contactEngine(restFactoryUri, REST.HttpConn.GET_METHOD)).asMap());
 
-            return new ConfiguredFactory(computer, FactoryBuilder.getFactory(keyspace, engineUrl, properties));
-        } catch (IOException e) {
-            throw new IllegalArgumentException(ErrorMessage.CONFIG_NOT_FOUND.getMessage(engineUrl, e.getMessage()));
+        String computer = null;
+        if(properties.containsKey(COMPUTER)){
```

Comment:
I think we can actually remove this check now. There should always be a computer I believe.